data_node/thredds.py

```diff
@@ -251,10 +251,11 @@ def write_tds_env():
 
 def update_mail_admin_address():
     '''Updates mail_admin_address in threddsConfig.xml'''
-    mail_admin_address = esg_property_manager.get_property("mail.admin.address")
-    esg_functions.stream_subprocess_output('sed -i "s/support@my.group/{mail_admin_address}/g" /esg/content/thredds/threddsConfig.xml'.format(mail_admin_address=mail_admin_address))
-
-
+    try:
```

Comment:
Proposed update  to avoid using subprocesses when possible. Sometimes I left subprocesses in when I either didn't understand what it was accomplishing initially or there was no direct analog to a python function.

`try:
        mail_admin_address = esg_property_manager.get_property("mail.admin.address")
    except ConfigParser.NoOptionError:
        return
    else:
        esg_functions.replace_string_in_file("/esg/content/thredds/threddsConfig.xml", "support@my.group", mail_admin_address)`

Reply:
Sorry that the new line are getting garbled in the proposed code block

Reply:
@William-Hill I made the change. Good catch